### PR TITLE
golang: Make this fast

### PR DESCRIPTION
**Pull request by purpleidea**
As per https://ttboj.wordpress.com/2017/02/26/faster-golang-builds/

### CODE REVIEW

**Comment by obnoxxx** [comment]
@purpleidea thanks for your patch.
I think i had wanted to add this "-i" to the build commands but had forgotten it again...
But on the (patch-)cosmetic side:

could you please reword the commit message? "golang: Make this fast" --> "build: make the compilation faster" or similar?

File: Makefile [hunk 1]
```diff
@@ -69,13 +69,13 @@ endif
 	glide install -v
 
 	echo "Building dependencies to make builds faster"
-	go install github.com/heketi/heketi
+	#go install github.com/heketi/heketi
```
Comment (obnoxxx) [inline Makefile]: 1. Why comment out and not remove?
2. AFAICT, this is not about making the build fast, but about removing something that you think is harmful. Could you please create a second patch for it?

File: Makefile [hunk 2]
```diff
@@ -69,13 +69,13 @@ endif
...
 
 glide.lock: glide.yaml
 	echo "Glide.yaml has changed, updating glide.lock"
 	glide update -v
 	echo "Building dependencies to make builds faster"
-	go install github.com/heketi/heketi
+	#go install github.com/heketi/heketi
```
Comment (obnoxxx) [inline Makefile]: dito

**Comment by obnoxxx** [comment]
I still think that this should have been 2 patches, as I said in my requests...
But I will be slightly generous on this one.... ;-)

**Comment by purpleidea** [comment]
@obnoxxx Oh, sorry-- I thought my improved commit message made it clear why I thought they were the same? If you want me to send a branch for a revert + two patches, LMK. More points that way.

**Comment by obnoxxx** [comment]
@purpleidea no need to say sorry. It's ok like this. Let's move on. ;-)

**Pull request merged**